audmath/core/api.py

```diff
@@ -98,6 +98,73 @@ def db(
     return x
 
 
+def distance(
+    u: typing.Union[typing.List, typing.Tuple, np.ndarray],
+    v: typing.Union[typing.List, typing.Tuple, np.ndarray],
+    *,
+    metric: str = 'cosine',
```

Comment:
I would use 'euclidean' as default metric, as I think that it is more general and more often used than cosine distance.

Even if we would later use the cosine distance for similarity measurement.

Reply:
So you would select `euclidean` as default for `distance()` and `cosine` for `similarity()` as these are the commonly used ones, correct?

Reply:
Yes, correctly, I don't have a strong opinion to use cosine similarity, but i would say this is the most practical choice.